modules/lx-autoscale/watchmaker-lx-autoscale.template.cfn.yaml

```diff
@@ -880,17 +893,25 @@ Resources:
               mode: '000400'
               owner: root
             /etc/cfn/scripts/watchmaker-install.sh:
-              content: !Sub |
-                  #!/bin/bash
-                  PYPI_URL=${PypiIndexUrl}
+              content:
+                !If
+                  - UseWamStandalone
+                  - !Sub |
+                      #!/bin/bash
+                      curl --fail -sSL -o /etc/cfn/scripts/watchmaker ${WatchmakerStandaloneUrl}
```

Comment:
What's the rationale behind putting the binary in /etc/cfn/scripts vice /usr/local/bin?

Reply:
Mostly just because this way it ends up in basically the same place on linux and windows...

Reply:
It would be nice to have watchmaker located within default paths.  I think the rough equivalent location on Windows would be C:\Program Files\Watchmaker.  What do you think about that approach?

Reply:
It definitely would be misleading on Windows without an actual native installer. This is not an installer at all, and watchmaker is not installed as far as Windows is concerned. This is just an executable file.